test/formatters/vsoFormatterTests.ts

```diff
@@ -45,7 +45,7 @@ describe("VSO Formatter", () => {
             getFailureString(TEST_FILE, 2, 12, "mid failure", "mid-name") +
             getFailureString(TEST_FILE, 9,  2,  "last failure", "last-name");
 
-        const actualResult = formatter.format(failures);
+        const actualResult = formatter.format(failures, failures);
```

Comment:
this is confusing to read -- the second argument is `fixes` but it's called `failures` here. you might want to write a separate test to validate that passing in fixes as a second argument doesn't create excess formatter output, or at least leave a comment here

Reply:
I created a separate test where I duplicate the failures array to pass them as `fixes`